product-service/getProductsById.js

```diff
@@ -0,0 +1,36 @@
+module.exports.getProductsById = async (event) => {
+  const index = Math.random() >= 0.5 ? 1 : 0;
```

Comment:
I guess it would be better to use id, I mean there are direct task for that, but I guess it supposed:
```
    const { productId } = event.pathParameters;
    const products = await getProducts()
    const product = products.find(({ id }) => id === productId)
    ```